### ISSUE TITLE
Feature Request: News to Facebook/Twitter

### ISSUE THREAD

**Issue by LaocheXe** [e107inc/e107#947]
So when you submit news it post it as a comment on facebook, twitter, social media - with news name, short description, and a link to the news item.

**Comment by CaMer0n** [comment]
Yes, this was a planned feature. (see the news tab, although it may not show unless in developer mode)

**Comment by LaocheXe** [comment]
I notice the Ping Service - have no clue about that, then in the Advance Tab is the post to twitter and facebook. no function on my end however - all I have a links to twitter and facebook - I take it I need to find what the ping services does.

**Comment by LaocheXe** [comment]
On the News page (news.php) each news post has the social media bar for users/guest to share the news post on their time line/twitter feed - issues is with the facebook buttons - the like button is weird - opens up in a new window with a white page with a blue like button up in the top left side - https://www.facebook.com/plugins/like.php?href=http%3A%2F%2Fdefiantz.org%2Fnews.php%3Fextend.5

The share goes to a white page - https://www.facebook.com/sharer/sharer.php?u=http%3A%2F%2Fdefiantz.org%2Fnews.php%3Fextend.5&t=Battlefield+Hardline+Multiplayer+Review

I don't know if facebook did another update to their code or what but the twitter one works.

**Comment by Jimmi08** [comment]
@LaocheXe use 3rd part service for this and rss plugin.  I use Zapier but there are more tools
  
@Moc This could be closed. With neverending changes from FB side this shouldn't be in core.  Fb_like menu is not working now, not something like this.
